### ISSUE TITLE
Bug: длинный урл ломает верстку

### ISSUE THREAD

**Issue by fey** [Hexlet/hexlet-correction#191]
### Summary

таблица не влезает в колонку, если урл очень длинный

![image](https://github.com/Hexlet/hexlet-correction/assets/697178/f24e2033-e6b8-40a2-adaa-58bcf4303681)

### Reproduction steps

```bash
Добавить опечатку с урлом, типа `https://guides.hexlet.io/ru/bootstrap/?_gl=1*uafu67*_ga*MTAwNjg4MjEyNC4xNjg5MzIxNTQw*_ga_PM3R85EKHN*MTY5MDU0NjAzOC4xMTQuMS4xNjkwNTQ2MDk4LjAuMC4w*_ga_WWGZ6EVHEY*MTY5MDU0NjAzOC4xMTMuMC4xNjkwNTQ2MDM4LjYwLjAuMA..` (и более)
```

### Expected result

Верстка выглядит корректно, в две колонки

### Actual result

На скриншоте.

### Browsers

Chrome

### OS

Linux

**Comment by bazilval** [comment]
Может имеет смысл ещё и срезать query params? По факту они не имеют смысла в рамках наших задач (виджет будет вешатьтся на всю страничку, а не страничку с query params)

**Comment by fey** [comment]
@bazilval параметры могут использоваться на страницах в форме. Представьте что у вас 10к записей (тех же опечаток), и пользователь (например в интернет магазине) прислал опечатку. Если вы просто глянете на страницу, то не найдете в чем дело. Поэтому важно нам иметь доступ к странице и открывать ее. 

Скорее всего нужно делать обрезку обрезку ссылки или поправить верстку. Но ссылку оставлять оригинальную. Плюсом думаю должна быть кнопка типа "перейти по ссылке опечатки"

**Comment by bazilval** [comment]
@fey мне казалось, что виджет нам отдаёт фактическую ссылку, когда делает пост-запрос для публикации репорта. Оттуда мы и можем взять актуальную для конкретной опечатки ссылку и переходить по ней.

А здесь речь об общей ссылке, характеризующей страницу, куда добавлен виджет.

**Comment by fey** [comment]
Не совсем понял. По моему там отображается урл, с которого отправили опечатку.

**Comment by bazilval** [comment]
@fey а, только теперь увидел, что речь именно про ссылку в опечатке

я просто смотрел на ссылку, которая в воркспейсе и там тоже вёрстка едет некрасиво, когда длинная ссылка
![image](https://github.com/Hexlet/hexlet-correction/assets/29205112/fbf003ce-bf92-4b59-940d-4c80e06259c0)

**Comment by bazilval** [comment]
@fey тогда думаю можно сделать так, что в таблице с опечатками url отображаются сокращёнными с многоточием на конце и при этом являются активными ссылками, чтобы можно было сразу перейти

возможно даже вообще имеет смысл скрыть ссылку, чтобы просто была активная ссылка "Ссылка"

**Comment by bazilval** [comment]
@fey Ну я через вёрстку и предлагаю, просто ссылке можно навесить класс Bootstrap "text-truncate" и style="max-width:450px;", получится вот так:
![image](https://github.com/Hexlet/hexlet-correction/assets/29205112/6dc24ded-8107-4ca0-abbd-debca0df84c3)

**Comment by fey** [comment]
@bazilval ок, давайте. Только почему 450px? 
Еще, я бы показал ПР (когда появится) верстальщикам Хекслета, чтобы посмотрели и дали совет как лучше сделать.

**Comment by bazilval** [comment]
Ну вот с этим надо ещё решить, да
Пусть верстальщики скажут как лучше
Я отталкивался от того, что при масштабе 100% это максимальная ширина, при которой не переносятся остальные колонки

**Comment by fey** [comment]
> Я отталкивался от того, что при масштабе 100% это максимальная ширина, при которой не переносятся остальные колонки

да, но это же от ширины экрана зависит?) У меня например есть второй монитор, он почти квадратный)

**Comment by bazilval** [comment]
Резонно)
Ну тут уж надо просто чтобы люди более разборчивые посмотрели

**Comment by bazilval** [comment]
@fey назначишь на меня?

**Comment by fey** [comment]
Актуально.

**Comment by shamshaev** [comment]
@fey Probably I fixed this bug. Please, check the screenshot. Now a long url doesn't break layout. In case of long text there the line just breaks to the next line.

![Screenshot from 2024-09-24 17-25-55](https://github.com/user-attachments/assets/080c1cd1-d414-4f2b-b519-48fb47e0c40d)

**Comment by fey** [comment]
@shamshaev good, go create PR =)
